frontend/src/interfaces/news.interface.ts

```diff
@@ -6,5 +6,5 @@ export type News = {
   published: boolean;
   target: string;
   created_at: string;
-  image_url: string;
+    image_url: string;
```

Comment:
reverse modification
```suggestion
  image_url: string;
```